### DISCUSSION TITLE
Is anybody else having troubles with new git-aggregator in 16.0 and 17.0?

### DISCUSSION THREAD

**Discussion by ap-wtioit** [Tecnativa/doodba#593] [Doodba - The image]
This not yet a reproducable (with public commits) issue but we are seeing this on more and more systems and today it also started affecting our dev systems.

After checking out a repo with doodba 16.0 / doodba 17.0 working with it and refreshing it with setup devel on the next day we get messages like this:
```
fatal: missing blob object '9f4995xxxxxxxxxxxxxx'
error: remote did not send all necessary objects
```

Only solution that helps here is to delete the whole repo (not good if developer has local branches that are not pushed yet) and check out again.

Is this an issue that only our version has or is somebody else affected?

**Comment by yajo** [comment]
I haven't seen this with git-aggregator, but I've seen it with mrchef, which is similar. Also I've had some cases of git checkout errors, but I've somehow fixed them all and moved on, since they weren't clearly reproducible.

**Comment by pedrobaeza** [comment]
Not experimenting it, but my development flow doesn't involve doing `git-aggregate` frequently.
